packages/backend/src/event/services/recur/util/recur.util.ts

```diff
@@ -69,13 +70,25 @@ const _generateInstances = (
   const instances = dates.map((date) => {
     const { startDate, endDate } = _getDates(rule, date);
 
+    // Adds original start/end hours to the instance
```

Comment:
Instances weren't capturing the hour of the original base event. This made them show up incorrectly on the UI. I added this bit, but I'm open to refactor. Overall, I feel the recurring logic needs some work to cover all the different rules.